Helidon 0.10.0 build failed in local system

## Environment Details

* Helidon Version: 0.10.0
* Helidon SE or Helidon MP
* JDK version: java version "1.8.0_181"
* OS: Windows 10
* Docker version (if applicable):

----------

## Problem Description

The **quickstart-se** and **quickstart-mp** modules defined in root pom.xml does not existed.

## Steps to Reproduce

1. Downloaded the source codes archive of 0.10.0 from **Release** tab
2. Extracted the files into local disc
3. Entered the root folder, and ran `mvn clean install`, and got the following errors:

```
[ERROR]
[ERROR]   The project io.helidon:helidon-project:0.10.0 (E:\Users\hantsy\temp\helidon-0.10.0\pom.xml) has 2 errors
[ERROR]     Child module E:\Users\hantsy\temp\helidon-0.10.0\quickstart-se of E:\Users\hantsy\temp\helidon-0.10.0\pom.xml does not exist
[ERROR]     Child module E:\Users\hantsy\temp\helidon-0.10.0\quickstart-mp of E:\Users\hantsy\temp\helidon-0.10.0\pom.xml does not exist

```

--- a/examples/archetypes/test-archetypes.sh
+++ b/examples/archetypes/test-archetypes.sh
@@ -24,8 +24,16 @@ else
   SCRIPT_PATH="${0}"
 fi
 
-# Current directory
+# Directory this script resides in
 MY_DIR=$(cd $(dirname -- "${SCRIPT_PATH}") ; pwd -P)
+# Current directory
+CURRENT_DIR=${PWD}
+
+# cd to an innocuous directory since the archetypes will create
+# a project here when we test it. See issue #64
+TARGET_DIR=${MY_DIR}/target
+mkdir -p ${TARGET_DIR}
+cd ${TARGET_DIR}
 
 ${MY_DIR}/create-archetypes.sh
 
@@ -60,3 +68,11 @@ mvn archetype:generate -DinteractiveMode=false \
 
 # build the generated project
 mvn -f ${PWD}/quickstart-mp/pom.xml install
+
+# Paranoia. Don't want to delete /!
+if [ ! -z "${TARGET_DIR}" ]; then
+    rm -rf ${TARGET_DIR}/
+fi
+
+# cd back to original directory 
+cd ${CURRENT_DIR}

